package.json

```diff
@@ -132,6 +122,7 @@
     "@types/react": "18.3.3",
     "@types/redux-logger": "3.0.13",
     "@types/sinon": "17.0.3",
+    "@vitejs/plugin-react": "4.3.1",
     "asyncbox": "3.0.0",
     "chai": "4.4.1",
```

Comment:
do we still use chai...?

Reply:
No, but there are several other dependencies that can also be removed, so I'd rather keep this PR smaller and remove all other dependencies in a separate one

Reply:
I can observe some test files already have `should` being replaced with `expect`.

Reply:
Yes, unit and integration tests are now fully based on `vitest`, and only the E2E test files still use `chai`. But again, I don't recall them ever working in the first place, so I'm thinking of just commenting out their contents and removing `chai` completely. Not in this PR though.

Reply:
fine with me